runtime/commands/fun.js

```diff
@@ -62,12 +62,14 @@ Commands.fortunecow = {
   timeout: 20,
   level: 0,
   fn: function (msg) {
-    unirest.get('https://thibaultcha-fortunecow-v1.p.mashape.com/random')
-      .header('X-Mashape-Key', config.api_keys.mashape)
-      .header('Accept', 'text/plain')
-      .end(function (result) {
-        msg.reply('```' + result.body + '```')
-      })
+    request.get('https://fortunecow.dougley.com/')
+    .end((err, result) => {
+      if (!err && result.statusCode === 200) {
```

Comment:
Detect `429 Too Many Requests` specifically and send an error to the channel asking them to wait.